cfgov/processors/wordpress_contact.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import json
+import os.path
+import requests
+
+
+def posts_at_url(url):
+
+    current_page = 1
+    max_page = sys.maxint
+
+    while current_page <= max_page:
+
+        url = os.path.expandvars(url)
+        resp = requests.get(url, params={'page': current_page})
+        results = json.loads(resp.content)
+        current_page += 1
+        max_page = results['pages']
+        total = 0
+        for p in results['posts']:
+            total += 1
+            yield p
+
+
+def documents(name, url, **kwargs):
+
+    for post in posts_at_url(url):
+        yield process_contact(post)
+
+
+def process_contact(contact):
+    del contact['comments']
+    contact['_id'] = contact['slug']
+
+    names = ['email', 'phone', 'fax']
+    for name in names:
+        if name in contact['custom_fields']:
```

Comment:
It looks like L:37-L:64 could also be:
```
if name in contact['custom_fields']:
…
elif name is 'fax':
…
else:
…
```

That'd be the same, right? It would remove:
```
if
else
    if
    else
```